lib/suse/connect/api.rb

```diff
@@ -84,6 +107,21 @@ def forward_system_deregistration(scc_system_id)
 
       protected
 
+      def prepare_payload_for_bulk_update(systems)
+        mandatory_keys = %i[login password last_seen_at]
+
+        systems_hash = systems.collect do |system|
```

Comment:
I think here we need to switch the payload based on the condition what is actually changed since the last sync: 

scc_synced_at: nil => Include all system attributes
last_seen_at > scc_synced_at => Only send `mandatory_keys`
last_seen_at <= scc_synced_at => system shouldn't be included at all in sync

Reply:
The first case is the de facto case as we set `scc_synced_at` to `nil` whenever a system attribute is updated.